KioskApplication/src/main/java/entity/RequestEntity.java

```diff
@@ -273,6 +270,28 @@ public Request getRequest(String requestID){
         return request;
     }
 
+    public void updateRequest(String requestID, String nodeID, String assigner, String note,
```

Comment:
Needs javadoc headers. In IntelliJ if you type `/**` and press enter, it creates a javadoc header for you.